internal/pkg/cli/env_init.go

```diff
@@ -152,7 +152,7 @@ func (o *initEnvOpts) Execute() error {
 	}
 
 	// 2. Get the environment
-	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName)
+	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName, o.IsProduction)
```

Comment:
Why are we doing it this way? can we not use `ssm.GetEnvironment()`

Reply:
at this stage it is not even written in SSM store...This is to write in SSM store.

Reply:
oh i see 🤦‍♂ my bad!

How difficult would it be to modify the `templates/environment/cf.yml` template to accept a `IsProd` parameter that defaults to `"false"`? Modifying the `conf.ToEnv` seems to be a neater solution than passing around the bool.

If that's not possible, then I think I'd prefer:
```suggestion
	env, err := o.envDeployer.GetEnvironment(o.ProjectName(), o.EnvName)
	if err != nil {
		return fmt.Errorf("get environment struct for %s: %w", o.EnvName, err)
	}
	env.IsProduction = o.IsProduction
```